css/colourpicker.css

```diff
@@ -79,12 +108,19 @@
   width: 150px;
   height: 150px;
   cursor: crosshair;
-  background: linear-gradient(to right,#FFF,rgba(255,255,255,0));
 }
 .colourpicker .colourpicker-grid-inner {
   width: 100%;
   height: 100%;
+  background: linear-gradient(to right,#FFF,rgba(255,255,255,0));
+  filter:progid:DXImageTransform.Microsoft.gradient(gradientType=1,startColorstr=#FFFFFFFF,endColorstr=#00000000);
+}
+
+.colourpicker .colourpicker-grid-inner-2 {
+  width:100%;
```

Comment:
whitespace (you can run the CSS file through a CSS linter/beautifier instead of manually adding the spaces. I won't add this comment anymore)